nextflow.config

```diff
@@ -20,145 +20,142 @@ profiles {
 }
 
 params {
-    	profile = "standard"
+	profile = "cluster"
 	tool_outputs_dir ="${baseDir}/data/tools_outputs"
-    	label_analysis_outdir ="${baseDir}/data/tools_outputs"
-
-    // Specify input data. If data_download = 'False'
-    ref_10x_dir = ""
-    query_10x_dir = ""
-    unmelt_sdrf_ref = "" // un-melted metadata ref file
-    unmelt_sdrf_query = "" // un-melted metadata ref file
-    ref_markers= ""
-    
-    // links to download data 
-    data_download{
-        run = "True" // must be 'True' or 'False'      
-        expr_data_type = "normalised"
-        normalisation_method = "TPM"
-
-        ref_acc_code = "E-MTAB-6386"
-        ref_output_dir = "ref_data"
-        ref_num_clust = 11
-
-        query_acc_code = "E-MTAB-6386"
-        query_output_dir = "query_data"
-        query_num_clust = 11 
-    }
+	label_analysis_outdir ="${baseDir}/data/tools_outputs"
```

Comment:
@rubenchazarra  is just the tab width that is changed here?

Reply:
Yep, only the new data_import slot is added. The rest remains the same @a-solovyev12